diff -uNr QtXlsxWriter.orig/CMakeLists.txt QtXlsxWriter/CMakeLists.txt
--- QtXlsxWriter.orig/CMakeLists.txt	2017-06-24 12:01:03.418915642 +0300
+++ QtXlsxWriter/CMakeLists.txt	2017-06-24 12:13:16.874339862 +0300
@@ -30,6 +30,7 @@
     target_compile_features(QtXlsxWriter PRIVATE cxx_range_for)
 endif()
 
+set_target_properties(QtXlsxWriter PROPERTIES VERSION 0.3.0 SOVERSION 0)
 set_target_properties(QtXlsxWriter PROPERTIES DEBUG_POSTFIX "d")
 target_link_libraries(QtXlsxWriter ${Qt5Core_LIBRARIES})
 target_link_libraries(QtXlsxWriter ${Qt5Gui_LIBRARIES})
